assets/js/util/test/numFmt.js

```diff
@@ -0,0 +1,122 @@
+/**
+ * `numFmt` tests
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { numFmt } from '../';
+
+/**
+ * Sets the Site Kit locale to a (legacy) global variable.
+ *
+ * @since 1.7.0
+ * @private
+ *
+ * @param {string} langCode The locale to set Site Kit to use. E.g. `en-US` or `de-DE`.
+ * @return {Object} Site Kit configuration object.
+ */
+const setupGoogleSiteKit = ( langCode ) => {
+	return global._googlesitekitLegacyData = {
+		locale: langCode,
+	};
+};
+
+// Unlike normal space. ASCII 32
+// This `NO-BREAK SPACE` or HTML's `&nbsp;` is used by Intl.NumberFormat between unit and number. ASCII `194 160`
+const NO_BREAK_SPACE = ' ';
+
+describe( 'numFmt', () => {
+	afterEach( () => {
+		global._googlesitekitLegacyData = null;
```

Comment:
This would then only need to reset the `locale` rather than overwrite the whole object.